docs/src/main/sphinx/sql/refresh-materialized-view.rst

```diff
@@ -0,0 +1,28 @@
+=========================
+REFRESH MATERIALIZED VIEW
+=========================
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    REFRESH MATERIALIZED VIEW view_name
+
+Description
+-----------
+
+Refresh the data stored in the materialized view ``view_name``. Data is
```

Comment:
Refreshes

Reply:
Thats against the style of all the other language docs. Leaving as is.